mesonbuild/scripts/gtkdochelper.py

```diff
@@ -107,7 +107,7 @@ def build_gtkdoc(source_root, build_root, doc_subdir, src_subdirs,
     gtkdoc_run_check(scan_cmd, abs_out)
 
     if gobject_typesfile:
-        scanobjs_cmd = ['gtkdoc-scangobj'] + scanobjs_args + [gobject_typesfile,
+        scanobjs_cmd = ['gtkdoc-scangobj'] + scanobjs_args + ['--type=' + gobject_typesfile,
```

Comment:
@ensonic and @tp-m : I provided a second commit fixing the above typo (indeed I meant `--types`). Please let me know if I should use another approach to fix it.